python/ray/tests/kuberay/test_autoscaling_config.py

```diff
@@ -222,6 +225,20 @@ def _get_ray_cr_with_no_tpus() -> dict:
     return cr
 
 
+def _get_ray_cr_with_only_requests() -> dict:
+    """CR contains only resource requests"""
+    cr = get_basic_ray_cr()
+
+    for container in [
+        container
+        for group in [cr["spec"]["headGroupSpec"]] + cr["spec"]["workerGroupSpecs"]
```

Comment:
These two loops are hard to read

Reply:
Updated. How about the new one:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/aa685f3d-9bae-4db4-ade3-293b0aaa73d7" />